lca-cli/seedcreator/seedcreator.go

```diff
@@ -108,10 +108,6 @@ func (s *SeedCreator) CreateSeedImage() error {
 		s.log.Info("Seed cluster certificates backed up successfully for recert tool")
 	}
 
-	if err := utils.RunOnce("delete_node", common.BackupChecksDir, s.log, s.deleteNode, ctx); err != nil {
-		return err
-	}
-
 	_ = utils.RunOnce("wait_for_ovn_to_go_down", common.BackupChecksDir, s.log, s.waitTillOvnKubeNodeIsDown, ctx)
```

Comment:
We should delete this one too and lets delete deleteNode function itself and waitTillOvnKubeNodeIsDown

Reply:
But we tested without deleting this line.
Are we sure it will still work?

Reply:
So far we have been testing with all the [changes](https://github.com/openshift-kni/lifecycle-agent/pull/290/files) you currently see in this PR.